data/hai/hai reveal 5 philosophers.txt

```diff
@@ -123,20 +123,20 @@ mission "Hai Reveal [B02] Join the Fleet"
 		# This special security force has acquired outfits in the post-war condition that were surplus or black market to upgrade their Splinters.
 		# However, they haven't had the time to refit and test out new options for their Protectors or Vanguards.
 
-	npc accompany
+	npc
 		government "Republic"
-		system "Ultima Thule"
-		personality escort heroic opportunistic waiting
+		system "Mirfak"
+		personality escort heroic opportunistic
 		fleet "Human Wormhole Guards" 20
-	npc accompany
+	npc
 		government "Hai"
-		system "Ultima Thule"
-		personality escort heroic opportunistic waiting
+		system "Mirfak"
+		personality escort heroic opportunistic
 		fleet "Hai Wormhole Guards" 20
-	npc accompany
+	npc
 		government "Hai (Friendly Unfettered)"
-		system "Ultima Thule"
-		personality escort heroic opportunistic vindictive waiting
+		system "Mirfak"
+		personality escort heroic opportunistic vindictive
```

Comment:
In my tests, if it has "accompany," and any ships in the fleet die, the mission is never completed.

Reply:
I think its because we're not on the last version?
Accompany shouldnt have save in it soon if it isnt already the case, I think

Reply:
> In my tests, if it has "accompany," and any ships in the fleet die, the mission is never completed.

That would have been the case on your initial test because the feature was added and removed for 9.16, and then re-added after due to some issues with the first version.

Reply:
I got the fleet escort logic to work. The solution is to have the ships spawn at Rajak. Story-wise, they started at Ultima Thule and jumped one system while the player entered orbit.  Now the player is told to rendezvous at Moktar, which they can do even with a slow ship if they hurry. That system isn't too busy. The "various fleets are arriving" dialog at Moktar may not be right, depending on how slow they are, but the fleets should arrive very soon after if they haven't already.

I can't put "accompany" back in because the mission still fails sometimes when ships die. Maybe you could submit an issue for that?